faq/faq.rst

```diff
@@ -72,3 +72,10 @@ How does citus support count(distinct) queries?
 -----------------------------------------------
 
 Citus can push down count(distinct) entirely down to the worker nodes in certain situations (for example if the distinct is on the distribution column or is grouped by the distribution column in hash-partitioned tables). In other situations, Citus uses the HyperLogLog extension to compute approximate distincts. You can read more details on how to enable approximate :ref:`count_distinct`.
+
+In which situations may I add a uniqueness constraint to a distributed table?
```

Comment:
Maybe rephrase to "In which situations are uniqueness constraints supported on distributed tables?"